### ISSUE TITLE
Holding key on Keyboard unstable

### ISSUE THREAD

**Issue by Yyatiko** [koreader/koreader#5319]
* KOReader version: v2019.08
* Device: Kobo_pika

#### Issue

Hi,

I have an issue with the letter that are on either side of the keyboard on the second line, i.e. **key a** and **key l**

As soon as I hold one of those key to access the popup, the popup window appear and disappear very quickly and one of the letter are chosen **a with a wave** when I hold **a** and 
a **cross** when I hold **l** (lowercase L).

There is a way to workaround. If I press slightly to the right of the letter **a**, the popup window stays and I can chose a letter. My finger has to be precisely align in the middle of the popup window, where the **a** appears (slightly off the original **a**). It work also symmetrically with **l**

I guess it is because the popup window has to shift a little bit when it appears because it is on the edge of the screen. 

All the other key are fine. This happen in the text editor as well as dictionary lookup and other...

**Comment by Frenzie** [comment]
Allowing the popup to overflow is not necessarily better (besides overflowing & looking ugly) because it makes the other buttons harder to reach. Unfortunately that means it's not quite a viable solution to popup overflowed and then shift to a final position on middle hold release.

![Screenshot_20190905_085421](https://user-images.githubusercontent.com/202757/64318638-ceafb880-cfba-11e9-9581-cae2c2f4204b.png)

Btw, I implemented hold release to take your time more as a kind of built-in documentation so you can contemplate the options without having to keep holding. The most efficient means of typing is to swipe in a particular direction, but the results of the swipe directions would be unknown without something visual. The popup itself in turn can also behave like you might be used to on a phone OS like Android, where you keep holding. This is usually the second fastest. Hold, release, contemplate is a lot slower but I really hated having to keep holding on my Android keyboard to keep looking at the options. ;-)

The tl;dr of that a nicer workaround is to keep holding and then move slightly to the left/right.

**Comment by Yyatiko** [comment]
Hi Frenzie,

Thanks for your answer. This is great : **the swipe**. It makes it very easy to type anything. I think it would be worth mentioning in the wiki, if not already done.

**Comment by Frenzie** [comment]
@AlanSP1 Is that mentioned anywhere yet?

**Comment by AlanSP1** [comment]
Wasn't mentioned, but it is now: https://github.com/koreader/koreader/wiki/Extended-keyboard---swipe-to-get-new-characters

I added link to article here (at the moment at the end of list): https://github.com/koreader/koreader/wiki/Features-list

Hope my short explanation is OK what it actually is. Maybe if @Yyatiko comment how I explained it?

**Comment by Yyatiko** [comment]
@AlanSP1 

Hi,

I just revised your first link. I hope it is ok. 

Thanks

**Comment by AlanSP1** [comment]
Yes, of course, help is always welcome and I'm glad you're made things much clearer.

**Comment by hius07** [comment]
Seems can be closed.

**Comment by NiLuJe** [comment]
Oh, yeah, we papered over that a while back, thanks!
